### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Kong-Codes**

### CODE REVIEW

File: project-5/main.py [hunk 1]
```diff
@@ -137,19 +150,29 @@ def validation(self):
...
 
-        self.insert()
+    def data_type_validation(self):
+        """
+        This function validates the data type
+        """
+        for num in range(len(self.df)):
+            if (self.df['Company Name'][num] == str or
+                    self.df['Company Link'][num] == str or
+                    self.df['Company LinkedIn'][num] == str):
```
Comment (Bee0933) [inline project-5/main.py]: @Kong-Codes can you use the `type()` function to validate the data type like: `type(self.df['Company Name'][num]) == str`

File: project-5/main.py [hunk 2]
```diff
@@ -111,24 +126,22 @@ def insert(self) -> None:
                 raise InsertError("cannot insert into table") from e
 
     def validation(self):
```
Comment (Bee0933) [inline project-5/main.py]: lets change the function name to `validate_duplicate_columns`

File: project-5/main.py (hunk 1 above)
Comment (Kong-Codes) [reply project-5/main.py]: Done i used isinstance() instead

File: project-5/main.py (hunk 2 above)
Comment (Kong-Codes) [reply project-5/main.py]: Done i used table validation because the function adds constraints to the table

**Pull request merged**